package.json

```diff
@@ -35,8 +35,9 @@
     "css-loader": "~0.23.1",
     "del": "~2.2.0",
     "edx-pattern-library": "~0.12.5",
-    "eslint": "~2.13.1",
-    "eslint-config-edx": "~1.2.1",
+    "eslint": "^3.16.0",
```

Comment:
AFAIK we don't need an explicit ESLint dependency anymore now that we're on NPM 3 (although maybe the Yarn thing you found has implications for this?)